app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -0,0 +1,182 @@
+module UspsInPersonProofing
```

Comment:
I just moved this file and moved the class into a module. The namespace and class name have changed but otherwise this is unchanged

Reply:
Should move & rename the test file as well, from
```
spec/services/usps_in_person_proofer_spec.rb
```
to
```
spec/services/usps_in_person_proofing/proofer_spec.rb
```